src/api/lib/feature_switch/feature.rb

```diff
@@ -2,7 +2,7 @@ module Feature
   @perform_initial_refresh_for_user = true
 
   def self.active?(feature)
-    if User.current && (User.current.is_admin? || User.current.is_staff?)
+    if User.current && (User.current.is_admin? || User.current.is_staff? || User.current.in_beta?)
```

Comment:
Do we have a way to enable certain features for staff members, but not for beta users? That would be handy for the bootstrap feature. And if not, we wouldn't have to check for staff members anymore because they could just opt into the beta group.

Reply:
We don't have a way to do this.

Reply:
Not really! That would mean that we need to even more monkey patch the feature switch which I think we should avoid. You only have true/false as keys but not like beta, that would require some invasive patches. I also wouldn't drop staff for now.

For bootstrap I don't really see the problem as we agreed in the planning that we don't announce the public beta until we have a significant amount of pages migrated (e.g. packages and project pages) so that the user actually have something to try out and don't switch forth and back all the time.

Reply:
> And if not, we wouldn't have to check for staff members anymore because they could just opt into the beta group.

I mean we could discuss to drop admin & staff here so admins would have the possibility to not see e.g. bootstrap. However, this is something we first need to discuss and I would do in another PR. That potentially also needs a migration.

Reply:
I think what you say is true for all `Staff` too. The only thing we need `Staff` for is peek.

Reply:
> For bootstrap I don't really see the problem as we agreed in the planning that we don't announce the public beta until we have a significant amount of pages migrated.

There is already one user who [ran into this](
https://github.com/openSUSE/open-build-service/issues/5574#issuecomment-412333411) (hosting an OBS instance and having the bootstrap theme for his Admin user).
That's why I am asking.

Reply:
So please remove Admin and Staff here.

Reply:
It's also more straightforward for each user to decide if they are in or not.